### ISSUE TITLE
Add a "skip to next file" link at top of every file in a diff

### ISSUE THREAD

**Issue by jamonholmgren** [isaacs/github#724]
I often have to review pull request diffs that include large, generated JS files (several thousand lines long). I really don't care about the generated code, so I'd like to just skip to the next "real" file.

I would like a link in the header that says "Skip to next file". There are already links in the drop-down in the floating menu bar above, so essentially attaching those to each file header would be good.

<img width="1018" alt="screen shot 2016-07-19 at 11 20 27 am" src="https://cloud.githubusercontent.com/assets/1479215/16961780/d56aa906-4da3-11e6-8bd4-af027fd47f75.png">

Alternatively, put a "Skip to next file" in the floating menu bar.

**Comment by Mottie** [comment]
Maybe #671 is an alternative solution for you?

**Comment by jamonholmgren** [comment]
@Mottie yeah, that userscript will help in the interim. Collapsing is a fine alternative too. The Skip button may be easier to implement?

**Comment by Mottie** [comment]
Ok, a ["diff-links" userscript](https://github.com/Mottie/GitHub-userscripts/wiki/GitHub-diff-links) is now available. It will work along side the diff toggle userscript.

![github-diff-links](https://cloud.githubusercontent.com/assets/136959/17044228/cb7a4fd2-4f83-11e6-850a-d80829bcc676.gif)

**Comment by jamonholmgren** [comment]
That's amazing, @Mottie !! Thanks!
